backend/src/main/java/wooteco/retrospective/presentation/attendance/AttendanceController.java

```diff
@@ -1,16 +1,25 @@
 package wooteco.retrospective.presentation.attendance;
 
+import java.util.List;
+
+import javax.validation.Valid;
+
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+
 import wooteco.retrospective.application.attendance.AttendanceService;
+import wooteco.retrospective.domain.attendance.Time;
+import wooteco.retrospective.domain.member.Member;
+import wooteco.retrospective.presentation.dto.attendance.AttendanceByTimeResponse;
 import wooteco.retrospective.presentation.dto.attendance.AttendanceRequest;
 import wooteco.retrospective.presentation.dto.attendance.AttendanceResponse;
 
-import javax.validation.Valid;
-
 @RestController
 @RequestMapping("/api")
```

Comment:
```suggestion
@RequestMapping("/api/time")
```
`time` 이라는 경로가 반복되니 이와같이 해주고 메서드의 ~Mapping에서는 중복되는 경로를 제거해 주는게 어떨까요?

또한 REST 컨벤션에 의하면, 요청하는 리소스는 복수로 명시한다고 되어 있습니다.
```suggestion
@RequestMapping("/api/times")
```
와 같은 형태가 좋겠네요.

그리고 AttendanceController인데 리소스는 time으로 명시되어 있어서 불일치가 발생하는 것 같아요.

아래 postTime에서도, 시간을 post하는게 아니라 참석자를 post하는게 맞는 것 아닌가요? 그걸 위해 timeId와 memberId를 받은거구요.. 나머지 메서드들도 마찬가지로 다 참석자 관련 로직인데 Time으로 명시되어 있는게 이상합니다!

Reply:
명세에 따랐을 뿐입니다! 프론트랑 얘기해볼게영~